### PR TITLE
Fix qchem tutorials

### PR DESCRIPTION
**Pull request by ixfoduap**
Recent changes have been made to qchem functions, which are causing errors in existing tutorials.

This PR fixes those errors for two quantum chemistry tutorials. 

PR #231 makes a similar fix for the Intro to VQE tutorial

### CODE REVIEW

File: demonstrations/tutorial_quantum_chemistry.py [hunk 1]
```diff
@@ -51,12 +51,13 @@
 # In this example, we construct the electronic Hamiltonian of one of the most unique
 # molecules: water. We begin by reading the positions of the oxygen and hydrogen atoms. The
 # equilibrium geometry of water is read from the file :download:`h2o.xyz </demonstrations/h2o.xyz>`
-# and stored in a list containing the symbol and the Cartesian coordinates of each atomic
-# species:
+# and stored in lists respectively containing the symbol and the Cartesian coordinates of each
```
Comment (agran2018) [inline demonstrations/tutorial_quantum_chemistry.py:54]: ``symbols`` is a list but ``coordinates`` is an array. I would rephrase it as:
"The atomic symbols and the Cartesian coordinates of the atoms are stored in a list and a 1D array, respectively."

File: demonstrations/tutorial_quantum_chemistry.py [hunk 2]
```diff
@@ -260,13 +261,16 @@
 ##############################################################################
 # Finally, the :func:`~.pennylane_qchem.qchem.molecular_hamiltonian`
 # function is used to automate the construction of the electronic Hamiltonian using
-# the functions described above.
+# the functions described above. It takes as input the atomic symbols and their nuclear
```
Comment (agran2018) [inline demonstrations/tutorial_quantum_chemistry.py:264]:
```suggestion
# the functions described above. It takes as input the atomic symbols and the nuclear

```

**Review by agran2018 — CHANGES_REQUESTED** [review]
Thanks @ixfoduap. I have just suggested two minor changes.

File: demonstrations/tutorial_quantum_chemistry.py (hunk 2 above)
Comment (ixfoduap) [reply demonstrations/tutorial_quantum_chemistry.py:264]: I think "their" is correct here, as each atom has corresponding nuclear coordinates

File: demonstrations/tutorial_quantum_chemistry.py (hunk 1 above)
Comment (ixfoduap) [inline demonstrations/tutorial_quantum_chemistry.py:54]:
```suggestion
#  which retrieves the symbol and the Cartesian coordinates of each

```

File: demonstrations/tutorial_quantum_chemistry.py [hunk 3]
```diff
@@ -260,13 +262,16 @@
...
```
Comment (ixfoduap) [inline demonstrations/tutorial_quantum_chemistry.py:265]:
```suggestion
# the functions described above. It takes as input the atomic symbols and nuclear

```

File: demonstrations/tutorial_quantum_chemistry.py [hunk 4]
```diff
@@ -260,13 +262,17 @@
 ##############################################################################
 # Finally, the :func:`~.pennylane_qchem.qchem.molecular_hamiltonian`
 # function is used to automate the construction of the electronic Hamiltonian using
-# the functions described above.
+# the functions described above. It takes as input the atomic symbols and nuclear
+
+# coordinates, which can be read directly from the geometry file.
```
Comment (ixfoduap) [inline demonstrations/tutorial_quantum_chemistry.py:267]:
```suggestion
# the functions described above. It takes as input the atomic symbols and nuclear
# coordinates, which can be read directly from the geometry file.

```

**Review by agran2018 — APPROVED** [review]
Thanks @ixfoduap !

**Review by thisac — APPROVED** [review]
Thanks for fixing this @ixfoduap! 💯
